cmd/juju/backups/doc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package backups provides the juju backups command.
+// Backup of juju's state is a critical feature, not only for juju users
+// but for use inside juju itself. This is likewise the case for the
+// ability to restore previous backups.
```

Comment:
Juju doesn't do restore any more - you need to use the external jujud-restore tool. We should mention that and link to it.